open_source_template.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Clone the CIROH open source project template repository
+git clone https://github.com/sepehrkrz/CIROH-open-source-project-template.git
```

Comment:
I appreciate this file being included, given that it was missing in the other templates. Like was said in the other comments, this should work (after merge) if you replace `sepehrkrz` with `CIROH-UA` directly.